crates/core/migrations/2022-11-25-194809_create_collection_activities_table/up.sql

```diff
@@ -0,0 +1,11 @@
+create table if not exists collection_activities (
```

Comment:
Since we are deleting offers and listings the activity for nft and profiles isn't accurate. What do you think about calling this marketplace_activities.

For the profile query we query by either the buyer or seller
For nfts by the metadata
For collections by the collection id